Gemfile.lock

```diff
@@ -1,34 +1,35 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.5.1)
+    better_errors (2.7.1)
       coderay (>= 1.0.0)
       erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (11.0.1)
-    coderay (1.1.2)
-    commander (4.4.7)
+    byebug (11.1.3)
+    coderay (1.1.3)
+    commander (4.5.2)
       highline (~> 2.0.0)
-    database_cleaner (1.7.0)
-    diff-lcs (1.3)
+    database_cleaner (1.8.5)
+    diff-lcs (1.4.2)
```

Comment:
### [diff-lcs](https://github.com/halostatue/diff-lcs)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.3 → 1.4.2

[_~~change-log~~, [source-code](https://github.com/halostatue/diff-lcs)_]





<details>
<summary>Commits</summary>

A change of **28** commits. See the full changes on [the compare page](https://github.com/halostatue/diff-lcs/compare/v1.3...v1.4.2).

These are the first 10 commits:
- (148ffe2) [Small README fix](https://github.com/halostatue/diff-lcs/commit/148ffe26c27fd56a68159cbbcdb4bff81d76028a)
- (15f739c) [[CI] Test against Ruby 2.5](https://github.com/halostatue/diff-lcs/commit/15f739ca7e40e39fac295a780bca47b1e97d9d77)
- (8b32565) [Merge pull request #42 from nicolasleger/patch-1](https://github.com/halostatue/diff-lcs/commit/8b325657b09ff2e382d626deed38beb02731f96a)
- (01e0cae) [Add #to_ary to Diff::LCS::Change and Diff::LCS::ContextChang…](https://github.com/halostatue/diff-lcs/commit/01e0caeda9ae38b7868f35987844fd9f67aef005)
- (3b4d2be) [Mention in rdoc that Diff::LCS::ContextChange can be convert…](https://github.com/halostatue/diff-lcs/commit/3b4d2be9c32041ebc2a567dcd6518c6fcf9908a8)
- (808f83a) [Make travis test the right things...](https://github.com/halostatue/diff-lcs/commit/808f83a11a9e6ec45b7ef2b9c336f4b75b9d8368)
- (07ed577) [Revert PR#47 "Add #to_ary to Diff::LCS::Change…"](https://github.com/halostatue/diff-lcs/commit/07ed577eba341f0ff0d7eebf4c1c2cc23083cba7)
- (45ea1b3) [Applied Rubocop rules that I like](https://github.com/halostatue/diff-lcs/commit/45ea1b30c7de0ad840f72afb82fd41eb2b9590db)
- (a798f6f) [Reintroduce Diff::LCS::Change#to_ary](https://github.com/halostatue/diff-lcs/commit/a798f6f1dc5d41aceb236ea0aeaa113c8dc89c92)
- (3a89de0) [Update documentation](https://github.com/halostatue/diff-lcs/commit/3a89de07745fea52f611e6955f61c11ffd68c754)


</details>